www/js/app.js

```diff
@@ -1152,6 +1181,11 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
             if (regexpActiveContent.test(htmlArticle)) uiUtil.displayActiveContentWarning();
         }
 
+        if(! isDirEntryExpectedToBeDisplayed(dirEntry)){
```

Comment:
This should be at the very beginning of `displayArticleContentInIframe`.
Else we could be in a situation where we display a warning about an article that we did not display.